Libs/vtkSegmentationCore/vtkSegmentationModifier.cxx

```diff
@@ -198,6 +198,24 @@ void vtkSegmentationModifier::SeparateModifiedSegmentFromSharedLabelmap(vtkOrien
     }
 }
 
+//-----------------------------------------------------------------------------
+void vtkSegmentationModifier::GetExtentUnion(vtkOrientedImageData *labelmap, const int extent[6], int extentUnion[6])
```

Comment:
The name suggests that this method computes union of the extents, but the implementation computes the intersection.  Did you intend to compute the intersection?

Reply:
You are correct, I got the 2 mismatched. I meant "intersection" :-)

Reply:
I have renamed the method to GetExtentIntersection instead.